srcpkgs/cropgui/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'cropgui'
 pkgname=cropgui
-version=0.3
+version=0.4
 revision=1
 hostmakedepends="python"
 makedepends="python"
```

Comment:
please change to py3 if possible, upstream's history indicates it support is already